packages/eslint-config/index.js

```diff
@@ -1,4 +1,12 @@
-const prettierConfig = require('./prettier.config');
+// The following are ESLint rules that collide with our Prettier config. If we don't disable these
+// then code formatted by Prettier will cause ESLint to throw errors.
+const prettierOverrides = {
+  'arrow-body-style': 'off',
+  'arrow-parens': 'off',
+  'object-curly-newline': 'off',
+  'operator-linebreak': 'off',
```

Comment:
these three appear to be in this list: https://github.com/prettier/eslint-config-prettier/blob/main/index.js

is `eslint-config-prettier` being loaded? i don't see it here